nss-day-cohort-22/Class-Website
# Replaced group photos to higher rez versions

## Link to Ticket

no direct ticket

## Description of Proposed Changes

- replaced Group photos with higher resolution versions

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout gl-replacingGroupPhotos

```
Run http-server
Check that you see a 3 group photos on the landing page in the carousel and that they don't look blurry

## Impacted Areas in Application

List general components of the application that this PR will affect:

* images/CohortGroupPhotos
* index.html

## Mentions @username

Tag users that need to review this code
